src/dbus_api.rs

```diff
@@ -266,7 +274,7 @@ fn ok_message_items() -> (MessageItem, MessageItem) {
 }
 
 fn default_object_path<'a>() -> dbus::Path<'a> {
-    dbus::Path::new(DEFAULT_OBJECT_PATH).unwrap()
+    dbus::Path::new("/").unwrap()
```

Comment:
Yeah, when to use consts should probably go in the (to-be-written) style guide. Especially in this case where we are in a function that just wraps the const, and there's really no other valid value for the const other than the one, using a const seems unnecessary. Consts are also really best for when more than one place relies on a value, which is not the case for the method names below. There is a trade-off, since it becomes a little harder to see all the defined methods that used to be grouped in a single place, but I still tend to think this is a preferable approach.